src/magic.h

```diff
@@ -152,8 +152,8 @@ struct fake_spell {
     translation npc_trigger_message;
 
     fake_spell() = default;
-    explicit fake_spell( const spell_id &sp_id, bool hit_self = false,
```

Comment:
Is it intentional to remove all these `explicit` specifiers?

Reply:
I'll fix that, I'm not sure why that happened.